File: CourierAmexClient/src/app/features/company/models/commodity.model.ts
import { BaseEntity } from "@app/models";

export interface CommodityModel extends BaseEntity<number> {
  companyId?: number;
  code: string;
  description: string;
  customsDuty?: number;
  customsGct?: number;
  customsFee?: number

  companyName?: string;
}

export const newCommodityModel = {
  id: 0,
  companyId: 0,
  code: '',
  description: '',
  status: 2
};

File: CourierAmexAPI/CourierAmex/Models/CommodityModel.cs
namespace CourierAmex.Models
{
    public class CommodityModel : BaseEntityModel<int>
    {
        public required int CompanyId { get; set; }
		public required string Code { get; set; }
		public required string Description { get; set; }
		public decimal CustomsDuty { get; set; }
		public decimal CustomsGct { get; set; }
		public decimal CustomsFee { get; set; }

		public string? CompanyName { get; set; }
    }
}
